assets/components/openshift-router/service-cloud.yaml

```diff
@@ -15,8 +15,6 @@ spec:
     - name: http
       port: 80
       targetPort: 80
-      nodePort: 30001
```

Comment:
Do we need the rebase script to automate this change?

Reply:
Yes, the node ports are added in the rebase script.
Updated the PR to remove the nodeport customization.

@fzdarsky  I noticed you added the node port customization in rebase script, is there any speical consideration of using nodeport 30001 and 30002?

Reply:
What is happening when we remove nodePort here

Reply:
@mangelajo It will allocate a random node port from the specified range.
This change may have side effect if we rely on hardcoded node port in microshift. I searched the repo and didn't find other reference to the fixed node port, but I'd like to double check.